### PR TITLE
disable triggering syncing agent sandbox entries into datomic

### PR DESCRIPTION
**Pull request by shamsimam**

### CODE REVIEW

File: scheduler/docs/configuration.adoc [hunk 1]
```diff
@@ -89,13 +89,12 @@ If need it to bind to another port, you can specify that with the `:local-port`
 `:sandbox-syncer`::
```
Comment (pschorf) [inline scheduler/docs/configuration.adoc]: I think the name `sandbox-syncer` isn't appropriate now that it's just a publisher.

File: scheduler/src/cook/mesos/sandbox.clj [hunk 2]
```diff
@@ -120,77 +120,32 @@
...
-                 datomic-db task-ids))
-          (remove-task-id-with-sandbox [task-id->sandbox task-id]
-            (dissoc! task-id->sandbox task-id))]
-    (->> (keys task-id->sandbox)
-         (retrieve-task-ids-with-sandbox (d/db datomic-conn))
-         (reduce remove-task-id-with-sandbox (transient task-id->sandbox))
-         persistent!)))
-
 (defn refresh-agent-cache-entry
   "If the entry for the specified agent is not cached:
```
Comment (pschorf) [inline scheduler/src/cook/mesos/sandbox.clj]: update comment

File: scheduler/src/cook/mesos/sandbox.clj [hunk 3]
```diff
@@ -212,63 +178,23 @@
...
         publisher-cancel-fn (start-sandbox-publisher
                               task-id->sandbox-agent datomic-conn publish-batch-size publish-interval-ms)
-        syncer-cancel-fn (start-host-sandbox-syncer publisher-state sync-interval-ms max-consecutive-sync-failure)]
+        hostname->task-id->sandbox-directory-fn (fn hostname->task-id->sandbox-directory-fn [agent-hostname task-id]
+                                                  (hostname->task-id->sandbox-directory
+                                                    publisher-state framework-id agent-hostname task-id))]
     (assoc publisher-state
-      :publisher-cancel-fn publisher-cancel-fn
-      :syncer-cancel-fn syncer-cancel-fn)))
+      :hostname->task-id->sandbox-directory-fn hostname->task-id->sandbox-directory-fn
```
Comment (pschorf) [inline scheduler/src/cook/mesos/sandbox.clj]: I really don't like the name of this function, is there something shorter that makes sense?

File: scheduler/docs/configuration.adoc (hunk 1 above)
Comment (shamsimam) [reply scheduler/docs/configuration.adoc]: Renamed to `sandbox-publisher`

File: scheduler/src/cook/mesos/sandbox.clj (hunk 2 above)
Comment (shamsimam) [reply scheduler/src/cook/mesos/sandbox.clj]: Updated.

File: scheduler/src/cook/mesos/sandbox.clj (hunk 3 above)
Comment (shamsimam) [reply scheduler/src/cook/mesos/sandbox.clj]: Renamed to `retrieve-sandbox-directory-from-agent`

**Pull request merged**